src/main/java/dk/cs/aau/huppaal/presentations/DropDownMenu.java

```diff
@@ -65,31 +68,20 @@ public DropDownMenu(final Node source, final int width, final boolean closeOnMou
 
         if (closeOnMouseExit) {
             final Runnable checkIfWeShouldClose = () -> {
-                if (!isHoveringMenu.get() && !isHoveringSubMenu.get()) {
-                    final Timer timer = new Timer(20, arg0 -> {
-                        if (!isHoveringMenu.get() && !isHoveringSubMenu.get()) {
-                            this.close();
-                        }
-                    });
-                    timer.setRepeats(false); // Only execute once
-                    timer.start(); // Go go go!
-                }
```

Comment:
Is the behavior okay without the timer adding a delay? I believe it was added in order to prevent the submenu from  closing prematurely

Reply:
I removed the timer, because 20 milliseconds is really not long enough to add any value.

Personally I would rather have it _only_ close when the user _clicks_ outside the dropdown menu and completely ignore hoverstate.

Reply:
I've made a different solution in f91b406559e728cc275b2b312d085f53343335b0

Reply:
I agree that it will be a better experience